ansible/configs/kni-osp/software.yml

```diff
@@ -0,0 +1,197 @@
+---
+- name: Step 001 software
+  hosts: utility
+  gather_facts: false
+  become: true
+  tasks:
+    - when: cloud_provider == "osp"
```

Comment:
You can remove this conditional and block. This config is only for OSP and I don't see anything in this play that would be different between cloud providers anyhow.